intelmq/bots/BOTS

```diff
@@ -1,5 +1,15 @@
 {
     "Collector": {
+
+        "bitsight":{
```

Comment:
This file is sorted, and no empty lines are allowed. You can use the `test_conf.py` script to validate the file.